pkg/views/profile/create.go

```diff
@@ -29,8 +29,8 @@ func ProfileCreationView(c *config.Config, profileAddView *ProfileAddView, editi
 			if str == "" {
 				return errors.New("profile name can not be blank")
 			}
-			if match, _ := regexp.MatchString("^[a-zA-Z0-9]+$", str); !match {
-				return errors.New("profile name must be alphanumeric only")
+			if match, _ := regexp.MatchString("^[a-zA-Z0-9._-]+$", str); !match {
```

Comment:
Let's just make this a constant in the `constants` package. It will be easier to reuse. We can call it `PROFILE_NAME_VALIDATION`

Reply:
done